.github/workflows/build.yml

```
@@ -109,9 +111,11 @@ jobs:
         run: dotnet tool restore
 
       - name: Run Uno Check to Install Dependencies
+        if: ${{ matrix.multitarget == 'wasm' || matrix.multitarget == 'ios' || matrix.multitarget == 'android' || matrix.multitarget == 'macos' || matrix.multitarget == 'uwp' }}
```

Would it be simpler to do an exclusion here instead?

---

It would be shorter and available multitargets aren't prone to change, so yes let's try that. I'm thinking we just invert the expression:
```suggestion
        if: ${{ matrix.multitarget != 'wasdk' && matrix.multitarget != 'linuxgtk' && matrix.multitarget != 'wpf' }}
```

Additional note: This isn't primarily a time saving strategy, this guard also allows us to pass our `matrix.multitarget` directly to the `--target` parameter for `uno-check` on the platforms where it's needed.

See [`uno-check` docs](https://platform.uno/docs/articles/uno-check.html#--target-choose-target-platforms) for a list of supported input values for this field.

---

You want to commit this to try then?